src/Illuminate/Cache/Repository.php

```diff
@@ -496,4 +496,28 @@ public function __clone()
     {
         $this->store = clone $this->store;
     }
+
+    /**
+     * Increment the value of an item in the cache.
+     *
+     * @param  string  $key
+     * @param  mixed  $value
+     * @return int|bool
+     */
+    public function increment($key, $value = 1)
```

Comment:
I don't really like just throwing methods at the bottom of a file. They have to go in a logical place within the class.